index.html

```diff
@@ -62,6 +76,30 @@ <h3>Let's Go For A Walk</h3>
 
     </section>
 
+    <section class="walk subscribe">
+      <div class="walk-img">
+        <img src="/img/magazine-pic.jpeg" alt="">
+      </div>
+
+      <div class="walk-text">
+        <h3>Stay Up To Date!</h3>
+
+        <p>Interested in space? Want to visit other planets or take a stroll through the stars?</p>
+
+        <p>Subscribe to SpaceWalkers Magazine, our weekly periodical that keeps you up to date on space exploration! We send out the magazine every week to your inbox. Subscribe to the right. It's quick and easy.</p>
+        
+        <div class="form">
+          <form action="">
+              <input type="text" name="firstname" value="First Name"><br>
```

Comment:
look into placeholder vs value

Reply:
Yes, I also worked on that last night. I was running out of time at the end of the end of the sprint. Basically I wanted to leave the value so it appears in the box and make the text opacity: 60% so that it tells the user what to put there, but not draw away from the rest of the design.